knotx-knot/knotx-knot-handlebars/src/main/java/io/knotx/knot/templating/impl/HandlebarsKnotProxyImpl.java

```diff
@@ -69,6 +91,41 @@ public HandlebarsKnotProxyImpl(HandlebarsKnotConfiguration configuration) {
     });
   }
 
+  private Template template(Fragment fragment) {
+    try {
+      byte[] cacheKeyBytes = digest.digest(fragment.content().getBytes(StandardCharsets.UTF_8));
+      String cacheKey = new String(cacheKeyBytes);
+
+      return cache.get(cacheKey, () -> {
```

Comment:
Could we log somewhere that cache capacity is exceeded (e.g. WARN).
That might be important, because if such scenario happens, there is not much use of this cache.

Reply:
```
    this.cache = CacheBuilder.newBuilder()
        .maximumSize(configuration.getCacheSize())
        .removalListener(listener -> LOGGER.warn("A cache limit is exceeded. See a 'cacheSize' configuration option."))
        .build();
```
Is it what you want?

Reply:
Yes, I guess this should be all right.
But I'd would change the message text to something like:
`"Cache limit exceeded. If this information occurs frequently, check 'cacheSize' configuration option because your cache is too small"`.

Reply:
done